packages/aws-cdk/lib/api/cxapp/cloud-assembly.ts

```diff
@@ -1,10 +1,10 @@
 import * as cxapi from '@aws-cdk/cx-api';
-import * as chalk from 'chalk';
 import { minimatch } from 'minimatch';
 import * as semver from 'semver';
-import { error, print, warning } from '../../logging';
+import { error, info, warning } from '../../logging';
 import { ToolkitError } from '../../toolkit/error';
 import { flatten } from '../../util';
+import * as chalk from '../../util/cdk-chalk';
```

Comment:
it seems we've previously made the decision to import from `../../util` (i.e. import from an `index.ts` file in the `util` folder). you are now doing something different for cdk-chalk. i am not opinionated on which way we do this, but i think it is important to be consistent, unless you have a compelling reason not to be. 

maybe it means that `cdk-chalk` doesn't belong in `util` at all?

the resolution to this comment is either turning it into `import { chalk, flatten} from '../../util'`, or moving `chalk` out of `util`, or providing a reason why it makes sense to import twice from `../../util` in this file.